src/recipes/build-release.ts

```diff
@@ -71,19 +71,17 @@ export const createReleaseBuildWorkflows = async (
   const existsIOsDir = toolbox.filesystem.exists('ios')
 
   if (expo) {
-    toolbox.print.info('⚙️ Running expo prebuild to setup app.json properly.')
-    await toolbox.interactive.spawnSubprocess(
-      'Expo prebuild',
-      `npx expo prebuild --${context.packageManager}`,
-      { alwaysPrintStderr: true }
-    )
+    toolbox.expo.prebuild(context, { cleanAfter: false })
   }
 
   const iOSAppName = toolbox.filesystem
     .list('ios')
     ?.find((file) => file.endsWith('.xcworkspace'))
     ?.replace('.xcworkspace', '')
 
+  if (!existsAndroidDir) toolbox.filesystem.remove('android')
```

Comment:
I'd add some comment or rename the variable - initially, I thought that you're removing folder if it doesn't exists - which odesn't make sense :D And what we're doing is we're removing the folder if user didn't had it/modified it before

Reply:
yes it is a bit misleading:D changed to `clean(Android/IOS)AfterPrebuild` in d2aad9439aab993dceaa7880f622a4f7a3f1b78e